afids_utils/tests/strategies.py

```diff
@@ -120,3 +120,24 @@ def affine_xfm(
         affine = matrices[xfm] @ affine
 
     return affine
+
+
+@st.composite
+def nii_img(
```

Comment:
The name of this suggests to me that it's going to produce a nibabel `Nifti1Image` (and by convention strategy names are pluralized). I'd call it `image_data_arrays` or something similar.